app/src/internal/java/com/kickstarter/InternalApplicationModule.java

```diff
@@ -65,11 +61,4 @@ static ApolloClientTypeV2 provideApolloClientTypeV2(final @NonNull ApolloClient
   static ApiClientTypeV2 provideApiClientTypeV2(final @NonNull ApiServiceV2 apiService, final @NonNull Gson gson) {
     return Secrets.IS_OSS ? new MockApiClientV2() : new ApiClientV2(apiService, gson);
   }
-
-  @Provides
-  @Singleton
-  @NonNull
-  static ApiClientType provideApiClientType(final @NonNull ApiService apiService, final @NonNull Gson gson) {
-    return Secrets.IS_OSS ? new MockApiClient() : new ApiClient(apiService, gson, true);
-  }
```

Comment:
are all of these deletions due to the resource shrinking?

Reply:
nop! this is the clean up noted, we should have removed that client a long time ago 

> Cleaned up lingering un-used ApiClient

it does messes up with the shrinking enabled so had to remove it